services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/local/BuildLocalStatisticsHelper.java

```diff
@@ -0,0 +1,238 @@
+   *
+   * @param federalStateCode - federal state code.
+   * @param localStatisticsJsonStringObject - local statistics json object.
+   * @return - Local Statistics supplier
+   */
+  public static Supplier<LocalStatistics> administrativeUnitSupplier(int federalStateCode,
+      LocalStatisticsJsonStringObject localStatisticsJsonStringObject) {
+    return () -> buildAdministrativeUnitStatistics(federalStateCode, localStatisticsJsonStringObject);
+  }
+
+  /**
+   * Enhances an instance of Local Statistics protobuf.
+   * Adds Administrative Unit Data to the provided Local Statistics
+   *
+   * @param federalStateCode - federal state code.
+   * @param localStatisticsJsonStringObject - local statistics json object.
+   * @return - Function which adds Administrative Unit Data to Local Statistics
+   */
+  public static Function<LocalStatistics, LocalStatistics> administrativeUnitEnhancer(int federalStateCode,
+      LocalStatisticsJsonStringObject localStatisticsJsonStringObject) {
+    return (localStatistics) ->
+        addAdministrativeUnitData(localStatistics, federalStateCode, localStatisticsJsonStringObject);
+  }
+
+  /**
+   * Get the federal state index used in configs and protobuf enum {@link FederalState}.
+   * It is the real federal state id subtracted by 1.
+   * This adjustment took place in order for the federal state to match the exact enum position.
+   * Protobuf enums must start from index 0.
+   *
+   * @param federalStateId - federal state id.
+   * @return - federal state index in configurations and protobuf enum.
+   */
+  public static int getFederalStateConfigIndex(Integer federalStateId) {
+    return federalStateId - 1;
+  }
+
+  /**
+   * Find trend enum from seven days incidence.
+   * INCREASING if trend = 1
+   * DECREASING if trend = -1
+   * STABLE if trend = 0
+   * @param sevenDayIncidence1stReportedTrend1Percent - seven day incidence
+   * @return - Trend
+   */
+  public static Trend findTrendBySevenDayIncidence(int sevenDayIncidence1stReportedTrend1Percent) {
+    switch (sevenDayIncidence1stReportedTrend1Percent) {
+      case -1:
+        return Trend.forNumber(3);
+      case 0:
+        return Trend.forNumber(1);
+      case 1:
+        return Trend.forNumber(2);
+      default:
+        return Trend.forNumber(-1);
+    }
+  }
+
+  /**
+   * Find federal state code from province code.
+   * For province codes containing 4 digits, the federal state code consists of first digit.
+   * For province codes containing 5 digits, the federal state code consists of first two digits.
+   * For province codes containing 1 or 2 digits, the federal state code is the the same as province code.
+   * @param provinceCode - province code.
+   * @return - federal state code.
+   */
+  public static Optional<String> findFederalStateByProvinceCode(String provinceCode) {
+    switch (provinceCode.length()) {
+      case 1:
```

Comment:
Is there a return statement missing?

Reply:
No, not really. Case 1 and 2 have the same return.